trade/trader_stakecube.js

```diff
@@ -428,19 +438,21 @@ module.exports = (
     async getDepositAddress(coin) {
       const paramString = `coin: ${coin}`;
 
-      let userData;
+      let scData;
       try {
-        userData = await stakeCubeApiClient.getUserData();
+        scData = await stakeCubeApiClient.getUserData();
 
-        if (userData.errorMessage) {
-          log.warn(`API request getDepositAddress(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${userData.errorMessage}`);
+        if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.